tests/system_tests/test_core/test_offline_sync.py

```diff
@@ -8,33 +8,34 @@
 
 
 @pytest.mark.flaky
-@pytest.mark.xfail(reason="flaky test")
-def test_sync_with_tensorboard(relay_server, runner, copy_asset, user):
+def test_sync_with_tensorboard(wandb_backend_spy, runner, copy_asset, user):
```

Comment:
```suggestion
def test_sync_with_tensorboard(wandb_backend_spy, runner, copy_asset):
```
i don't think you need the user fixture if you are using wandb_backend_spy